src/content/api/cli.md

```diff
@@ -70,49 +90,82 @@ If your project structure is as follows -
 ```
 
 ```bash
-webpack src/index.js -o dist/bundle.js
+webpack src/index.js -o dist
 ```
 
-This will bundle your source code with entry as `index.js`, and the output bundle file will have a path of `dist`, and the filename will be `bundle.js`
+This will bundle your source code with entry as `index.js`, and the output bundle file will have a path of `dist`.
 
 ```bash
 	| Asset     | Size    | Chunks      | Chunk Names |
 	|-----------|---------|-------------|-------------|
-	| bundle.js | 1.54 kB | 0 [emitted] | index       |
+	| main.js | 1.54 kB | 0 [emitted] | index       |
 	[0] ./src/index.js 51 bytes {0} [built]
 	[1] ./src/others.js 29 bytes {0} [built]
 ```
 
 ```bash
-webpack ./src/index.js ./src.index2.js -o dist/bundle.js
+webpack ./src/index.js ./src.index2.js -o dist/
 ```
 
 This will form the bundle with both the files as separate entry points.
 
 ```bash
 	| Asset     | Size    | Chunks        | Chunk Names   |
 	|-----------|---------|---------------|---------------|
-	| bundle.js | 1.55 kB | 0,1 [emitted] | index, index2 |
+	| main.js | 1.55 kB | 0,1 [emitted] | index, index2 |
 	[0] ./src/index.js 51 bytes {0} [built]
 	[0] ./src/index2.js 54 bytes {1} [built]
 	[1] ./src/others.js 29 bytes {0} {1} [built]
 ```
 
+## Default Configurations
+
+CLI will look for some default configurations in the path of your project, here are the config files picked up by CLI.
+
+If no `mode` is supplied via flags or config then this is the lookup order in increasing order
+
+> example - config file lookup will be in order of .webpack/webpack.config.development.js > webpack.config.development.js > webpack.config.js
+
+```md
+'webpack.config',
+'webpack.config.dev',
+'webpack.config.development',
+'webpack.config.prod',
+'webpack.config.production',
+'.webpack/webpack.config',
+'.webpack/webpack.config.none',
+'.webpack/webpack.config.dev',
+'.webpack/webpack.config.development',
+'.webpack/webpack.config.prod',
+'.webpack/webpack.config.production',
+'.webpack/webpackfile',
+```
+
+If `mode` is supplied, say `production` then config looking order will be -
 
-### Common Options
+`.webpack/webpack.config.production.* > .webpack/webpack.config.prod.* > webpack.config.production.* > webpack.config.prod.* > webpack.config.*`
+
+## Common Options
 
 W> Note that Command Line Interface has a higher precedence for the arguments you use it with than your configuration file. For instance, if you pass [`--mode="production"`](/configuration/mode/#usage) to webpack CLI and your configuration file uses `development`, `production` will be used.
 
-__List all of the options available on the cli__
+__List all of the commands and flags available on the cli__
 
 ```bash
 webpack --help
 webpack -h
 ```
 
+__Show help for a single command or flag__
+
+```bash
+webpack <command> --help
+webpack --<flag> --h
```

Comment:
Help on flags don't work
```suggestion
```

Reply:
should be `webpack --help <command>` or ``webpack --help <flag>``

Reply:
Yes, flag on help works.

Reply:
works both ways

Reply:
Nah, only flag on help works.